check_jobs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Check jobs acceptance timeout.
+# If jobs cannot be confirmed, listening will be terminated.
+
+if [ -z "$1" ]; then exit 1; fi
```

Comment:
this can never happen since we already ran `-n` on that env var which is passed here

```suggestion

```